iron-overlay-manager.html

```diff
@@ -22,6 +23,8 @@
 
     _backdrops: [],
 
+    _backdropElement: document.createElement('iron-overlay-backdrop'),
```

Comment:
Hmm there is a problem now, a circular dependency issue I think: the manager imports the backdrop and the backdrop imports the manager, I guess that's why I get `Uncaught TypeError: Cannot read property 'getBackdrops' of undefined`